x-pack/plugins/security_solution/public/management/pages/blocklist/view/blocklist.test.tsx

```diff
@@ -26,6 +32,13 @@ describe('When on the blocklist page', () => {
     act(() => {
       history.push(BLOCKLIST_PATH);
     });
+
+    mockedEndpointPrivileges = { canWriteBlocklist: true };
```

Comment:
Can you use `getEndpointAuthzInitialStateMock()` here to instead of setting this to a partial object? it will be easier to maintain in the future, especially if we start to check other RBAC properties for whatever reason - they will be already set.